qa/rolling-upgrade/src/test/java/org/opensearch/upgrades/IndexingIT.java

```diff
@@ -127,8 +130,7 @@ private void waitForSearchableDocs(String index, int shardCount) throws Exceptio
                     replicaRequest.addParameter("preference", "_shards:" + shardNum + "|_only_nodes:" + replicaShardToNodeIDMap.get(shardNum));
                     Response replicaResponse = client().performRequest(replicaRequest);
                     int replicaHits = ObjectPath.createFromResponse(replicaResponse).evaluate("hits.total");
-                    logger.info("--> ReplicaHits {}", replicaHits);
-                    assertEquals(primaryHits, replicaHits);
+                    assertEquals("Doc count mismatch for shard " + shardNum + ". Primary hits " + primaryHits + " Replica hits " + replicaHits, primaryHits, replicaHits);
```

Comment:
nit - assertion would already log primaryHits & replicaHits on a failure.

Reply:
The message adds shard number and in general better readable.